src/sqlfluff/dialects/dialect_athena.py

```diff
@@ -0,0 +1,540 @@
+"""The AWS Athena dialect.
+
+https://docs.aws.amazon.com/athena/latest/ug/what-is.html
+"""
+
+from sqlfluff.core.dialects import load_raw_dialect
+from sqlfluff.core.parser import (
+    AnyNumberOf,
+    BaseSegment,
+    Bracketed,
+    Delimited,
+    NamedParser,
+    Nothing,
+    OneOf,
+    OptionallyBracketed,
+    Ref,
+    RegexParser,
+    Sequence,
+    StringLexer,
+    StringParser,
+    SymbolSegment,
+)
+from sqlfluff.core.parser.grammar.anyof import AnySetOf
+from sqlfluff.core.parser.segments.raw import CodeSegment, KeywordSegment, RawSegment
+from sqlfluff.dialects import dialect_ansi as ansi
+from sqlfluff.dialects.dialect_athena_keywords import (
+    athena_reserved_keywords,
+    athena_unreserved_keywords,
+)
+
+ansi_dialect = load_raw_dialect("ansi")
+
+athena_dialect = ansi_dialect.copy_as("athena")
+
+athena_dialect.sets("unreserved_keywords").update(athena_unreserved_keywords)
+athena_dialect.sets("reserved_keywords").update(athena_reserved_keywords)
```

Comment:
You’re updating these without clearing them.

That’s OK, and will likely result in less errors as the ANSI piece of grammar you are not overriding will still work and will not require overriding to set them to `Nothing()`

However, the downside of this is all ANSI reserved words will still be reserved words in anthena. Only a few of these so probably not a problem.
But also all the unreserved words will be treated as unreserved and there’s a bigger list of those that will only be allowed to be used as identifiers if quoted.

It’s probably fine to leave as is (especially as an initial commit) and many of the dialects are done like this for simplicity reasons), but just wanted to make you aware of the downsides.

Reply:
Thanks a lot for clarifying! 
Going forward I will probably take some time to evaluate if we should clear at least the unreserved keywords.